content/quests/events/demevent_bog_vaccine.lua

```diff
@@ -17,129 +17,169 @@ QDEF:AddConvo()
-                    Anyway, just take some rest, and try not to work yourself too hard.
-                    You should be immune to bog parasites very soon.
-                    Anyway, see you!
+                    Bite your tounge. Whatever you feel now is leagues better than the eternal damnation at the hands of Hesh itelf.
+                    It will take it's toll, yes, but your body and soul will be fit for Consumption.
+                player:
+                    And how long will I be paying the fare for this mistake?
+                agent:
+                    !hesh_greeting
+                    As long as you keep fighting it.
                     !exit
-                * The priest leaves you be.
+                * You roll your sleeve back up and trudge on, feeling a noticable limp, but also a notable strength you can't quite place.
             ]],
             OPT_CONVINCE = "Question {agent} about the source of the vaccine",
             DIALOG_CONVINCE = [[
                 player:
-                    [p] Where did you find the vaccine?
-                agent:
-                    Excuse me?
-                player:
-                    This infestation wasn't in the pearl for long, but there is already a vaccine for it.
-                    Surely you don't find that suspicious?
+                    !crossed
+                    Pardon if a strange cocktail of chemicals being injected into me puts me on edge.
+                    I'd like to know what you put into this thing if you're going to give it to me.
             ]],
             DIALOG_CONVINCE_SUCCESS = [[
                 agent:
-                    [p] Will you shut up if I tell you?
-                player:
-                    Let's hear it.
-                agent:
-                    !sigh
-                    When we found out about the infestation, the cult starts to panic.
-                    Then, one of us noted that the Barons have worked in the bog for a long time, and surely they have a solution.
-                    So we "acquired" the vaccine from them.
+                    If it will help prevent the disease from spreading...
+                    Very well. We hadn't made the vaccine as immediately as we tell people.
+                    It was...supplied to us.
                 player:
-                    Uh huh.
+                    !question
+                    Supplied from where?
                 agent:
-                    Anyway, where it comes from doesn't matter.
-                    What matters is that you take the vaccine, lest you get infected by the bog.
+                    The Spark Barons had prepared an innoculative agent against it, although they hadn't made it particularly consumer friendly.
+                    !over_there
+                    But we had no choice. We had to protect the pious masses, so we purchased them and helped as many as we could.
```

Comment:
The original idea is that the barons have used the vaccine in the bog before on the workers and the barons, and that is why only Rook gets the parasite every time. Then when the boggers came to Pearl, the cult requisitioned it from the barons using underhanded means (either through stealing or forcing the barons to give them up). Either way the barons are not happy that the cult take their vaccine without paying them.
This information is then used as "dirt" allowing you to reject the vaccine without consequence (other than the eventual regret when you get a forced bog parasite down your throat).
I'm not sure whether this interpretation is better or worse. Either way take a second look.